include/openssl/dherr.h

```diff
@@ -8,10 +8,10 @@
  * https://www.openssl.org/source/license.html
  */
 
-#ifndef HEADER_DHERR_H
-# define HEADER_DHERR_H
+#ifndef OPENSSL_DHERR_H
+# define OPENSSL_DHERR_H
 
-# ifndef HEADER_SYMHACKS_H
+# ifndef OPENSSL_SYMHACKS_H
```

Comment:
Should the inclusion of headers be guarded like this or just included?  Not for this PR, just a general thought.

Reply:
See https://github.com/openssl/openssl/pull/9281#discussion_r299192179 and following...